Sources/Get/PackagesDirectory.swift

```diff
@@ -35,7 +34,7 @@ class PackagesDirectory {
 
         var result = Dictionary<String, Git.Repo>()
         for name in try! localFS.getDirectoryContents(self.prefix) {
-            let prefix = Path.join(self.prefix, name)
+            let prefix = self.prefix.appending(RelativePath(name))
```

Comment:
It would be nice to have a method for the case where you want to append a literal path component, that avoids the overhead of RelativePath normalization.